src/tasks/settings.html

```diff
@@ -0,0 +1,15 @@
+<h4>{{ 'task.settings.title' | translate }}</h4>
+<form id="tasks.settings.form" name="tasksSettingsForm" class="col-md-8">
```

Comment:
Please remove the `class` attribute to keep the look of the form consistent with the rest of the Motech UI forms